pop-api/examples/fungibles/tests.rs

```diff
@@ -319,15 +315,13 @@ fn transfer_from_works(mut session: Session) {
 	assert_eq!(session.sandbox().balance_of(&TOKEN, &ALICE), value);
 	assert_eq!(session.sandbox().balance_of(&TOKEN, &BOB), value);
 	// Successfully emit event.
-	assert_eq!(
-		last_contract_event(&session).unwrap(),
+	assert_last_event!(
```

Comment:
Perhaps the naming should remain `assert_last_contract_event!` as devs mind want to assert events in the runtime as well?

Reply:
Make sense!